clippy_lints/src/utils/conf.rs

```diff
@@ -459,6 +459,10 @@ define_Conf! {
     /// Whether to **only** check for missing documentation in items visible within the current
     /// crate. For example, `pub(crate)` items.
     (missing_docs_in_crate_items: bool = false),
+    /// Lint: LARGE_FUTURE
+    ///
+    /// The maximum size of a `Future` can have.
+    (future_size_threshold: u64 = 16 * 1024),
```

Comment:
Also, could you add a test for this config value inside the `tests/ui-toml/` test folder?